articles/automatic-software-install-in-fleet.md

```diff
@@ -0,0 +1,235 @@
+mechanism works.
+
+Fleet allows its users to upload trusted software installation files to be installed and used on hosts.
+This installation could be conditioned on a failure of a specific Fleet Policy.
+
+A very simple example will be this: 
+Install a patch on all MacOS hosts with version lower than 14.2.1.
+You will create a policy that assures hosts are equal or higher than 14.2.1 
+Like this: 
+- ```SELECT 1 where exists (SELECT version FROM os_version WHERE version >= "14.2.1");```
+
+Then all hosts failing this policy will have the patch programmed to be installed.
+
+Of course this feature holds a strong and flexible way to install software based on any chosen policy.
+See step by step section below.
+
+## Step-by-Step Instructions
+
+1. Add any software to be available for installation. Follow this document with instruction how to
+   do it.
+   Note that all steps of the software installation as set here will be respected regardless to the
+   policy that may trigger this software installations.
+   ('Pre-install query', 'Install script' and 'Post-install script').
+
+   TODO: Sharon - add link to SW install doc.
+
+![Add Software](../website/assets/images/articles/automatic-software-install-add-software.png)
+
+```
+Current supported installation files, manual upload of these formats:
+- Macos: .pkg
+- Windows: .msi, .exe
+- Linux: .deb
+
+Coming soon:
+- Ability to auto install from App store (VPP).
+- Install on iOS and iPadOS
+```
+
+
+2. In Fleet, add a policy that failure to pass it will trigger the required installation.
+  Go to Policies tab --> Press the top right "Add policy" button. --> Click "create your own policy"
+  --> Save --> Fill details in the Save modal and Save.
+
+![Add New Policy](../website/assets/images/articles/automatic-software-install-add-new-policy.png)
+
+3. Open Manage Automations: Policies Tab --> top right "Manage automations" --> "Install software".
+
+![Plocies Manage](../website/assets/images/articles/automatic-software-install-policies-manage.png)
+
+4. Select (click th echeck box of) your newly created policy. To the right of it select from the
+   drop-down list the software you would like to be installed upon failure of this policy.
+
+![Install Software Modal](../website/assets/images/articles/automatic-software-install-install-software.png)
+
+Upon failure of the select policy, the selected software installation will be triggered.
+
+
+
+
+
+## How does it work?
+
+* After configuring Fleet to auto-install a specific software the rest will be done automatically.
+* TODO Sharon: Describe the workflow 
+
+![Flowchart](../website/assets/images/articles/automatic-software-install-workflow.png)
+*Detailed flowchart*
+
+## Prerequisites
+
+* Fleet premium. 
+* Admin permissions for all three services above.
+
+
+
+
+
+## Additional Information
+
+* TODO: Add link to software install document
+* TODO: Add other docs
+
+
+### Using fleet API:
```

Comment:
@spokanemac, do we have a BKM for referring to our API/Fleetctl/Gitops instead of adding it here?

Reply:
Great! Thx.